catalog/app/components/BreadCrumbs/BreadCrumbs.js

```diff
@@ -29,6 +29,7 @@ export function copyWithoutSpaces(e) {
       .getSelection()
       .toString()
       .replace('<EMPTY>', '')
+      .replace('ROOT', '')
```

Comment:
It's not clear to me, what is going on here :)

I think that it's a good practice to avoid the use of strings in a such way. It's a sort of Magic Number pattern (https://en.wikipedia.org/wiki/Magic_number_(programming))
With named constants, it would be simpler to understand the intentions of code.

Reply:
well, i somewhat agree, but it's just how package root dir and empty prefixes show up in the breadcrumbs